addons/html_editor/static/tests/editor/qweb.test.js

```diff
@@ -0,0 +1,152 @@
+import { describe, expect, test } from "@odoo/hoot";
+import { click, queryAllTexts, queryFirst, queryOne, select } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { setupEditor } from "../test_helpers/editor";
+import { getContent } from "../test_helpers/selection";
+import { QWebPlugin } from "../../src/editor/qweb/qweb_plugin";
+import { registry } from "@web/core/registry";
+
+const config = { Plugins: [...registry.category("phoenix_plugins").getAll(), QWebPlugin] };
```

Comment:
i think you can just specify QWeb plugin, and the system will add all dependencies

Reply:
No it doesn t work